src/main/kotlin/io/github/mojira/arisa/modules/RemoveVersionModule.kt

```diff
@@ -17,10 +20,23 @@ class RemoveVersionModule(
                 .map { it.remove }
             assertNotEmpty(removeAddedVersions).bind()
             removeAddedVersions.forEach(::run)
+            if (calledMoreThan5Times(changeLog)) {
+                changeReporter("NoUser")
+            }
             addComment(CommentOptions(message))
         }
     }
 
+    private fun calledMoreThan5Times(changeLog: List<ChangeLogItem>): Boolean {

```

Comment:
I suggest renaming the function to remove the "5", as we may want to change the value in the future